### PR TITLE
100% tests coverage + bug fixes :-)

### PR DESCRIPTION
**Pull request by Izopi4a**

### CODE REVIEW

File: extension/sanitizer/ext/configure.ac [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+PHP_GEN_BUILD_DIRS
+PHP_GEN_GLOBAL_MAKEFILE
+
+test -d modules || $php_shtool mkdir modules
+touch .deps
+
+AC_CONFIG_HEADER(config.h)
+
+AC_OUTPUT()
```
Comment (sergeyklay) [inline extension/sanitizer/ext/configure.ac]: You should gitignore this file

File: extension/sanitizer/ext/php_sanitizer.h [hunk 2]
```diff
@@ -14,7 +14,7 @@
 #define PHP_SANITIZER_VERSION     "0.0.1"
 #define PHP_SANITIZER_EXTNAME     "sanitizer"
 #define PHP_SANITIZER_AUTHOR      ""
-#define PHP_SANITIZER_ZEPVERSION  "0.11.2-dd63df153e"
+#define PHP_SANITIZER_ZEPVERSION  "0.11.4-$Id$"
```
Comment (sergeyklay) [inline extension/sanitizer/ext/php_sanitizer.h:17]: Glad to see :)

File: extension/sanitizer/sanitizer/Sanitizers/Floats.zep [hunk 3]
```diff
@@ -30,8 +30,7 @@ class Floats extends Sanitizer implements SanitizerInterface, SizeInterface {
 
         let this->val = round(floatval(v), this->maxDecimals);
 
-
-        if (this->val === 0.00) {
+        if intval(this->val) === 0 {
```
Comment (sergeyklay) [inline extension/sanitizer/sanitizer/Sanitizers/Floats.zep:33]: `(int) this->val` is faster than `intval(this->val)`

Comment (Izopi4a) [reply extension/sanitizer/sanitizer/Sanitizers/Floats.zep:33]: :+1:

**Pull request merged**